src/comb/template.clj

```diff
@@ -18,23 +19,33 @@
         ")?"
         "(.*)\\z")))
 
-(defn emit-string [s]
-  (print "(print " (pr-str s) ")"))
+(defn emit-string
+  ([s]
+     (print "(print " (pr-str s) ")"))
+  ([s emit-next-line-break?]
+     (if emit-next-line-break?
```

Comment:
Good idea, but we need to decide how to deal with `<%=- %>`, ignore `-` or print result and trimming whitespace?